maubot/cli/commands/init.py

```diff
@@ -33,9 +33,9 @@ def load_templates():
     global mod_template, meta_template, base_config, loaded
     if loaded:
         return
-    meta_template = Template(resource_string("maubot.cli", "res/maubot.yaml.j2").decode("utf-8"))
-    mod_template = Template(resource_string("maubot.cli", "res/plugin.py.j2").decode("utf-8"))
-    base_config = resource_string("maubot.cli", "res/config.yaml").decode("utf-8")
+    meta_template = Template(resources.read_text("maubot.cli", "res/maubot.yaml.j2"))
+    mod_template = Template(resources.read_text("maubot.cli", "res/plugin.py.j2"))
+    base_config = resources.read_text("maubot.cli", "res/config.yaml")
```

Comment:
`read_text` is deprecated since Python 3.11.

```suggestion
    meta_template = Template(resources.files("maubot.cli").joinpath("res/maubot.yaml.j2").read_text(encoding="utf-8"))
    mod_template = Template(resources.files("maubot.cli").joinpath("res/plugin.py.j2").read_text(encoding="utf-8"))
    base_config = resources.files("maubot.cli").joinpath("res/config.yaml").read_text(encoding="utf-8")
```

Reply:
I committed your suggestion. It's part of the PR now.